pkgs/development/libraries/science/math/faiss/default.nix

```diff
@@ -47,6 +48,8 @@ stdenv.mkDerivation {
     pythonPackages.setuptools
     pythonPackages.pip
     pythonPackages.wheel
+  ] ++ lib.optionals stdenv.cc.isClang [
```

Comment:
I see that e.g. `igraph` is already using this pattern, but I'm wondering: is there a way to ensure that `stdenv` and `llvmPackages` are synchronized? Basically, to ensure that `stdenv = llvmPackages.stdenv`

Reply:
Not that I know... also, now we're in 22.11 freeze, so any stdenv changes will likely have to wait a few weeks.